ui/src/components/OrderModal.jsx

```diff
@@ -0,0 +1,32 @@
+import ParcelSizeSummaryCard from "./SummaryCards/ParcelSizeSummaryCard";
+import RecipientSummaryCard from "./SummaryCards/RecipientSummaryCard";
+import SenderSummaryCard from "./SummaryCards/SenderSummaryCard";
+import {Dialog} from "@material-ui/core"
+import "../styles/OrderModal.css"
+
+export default function OrderModal(props){
```

Comment:
Just a minor thing - missing space after `(props)` 😄

Reply:
@Zygis0321 do you have prettier installed?